utils/istio/istio.go

```diff
@@ -48,13 +49,33 @@ func GetIstioDestinationRuleGVR() schema.GroupVersionResource {
 	}
 }
 
+func GetVirtualServiceNamespaceName(vsv string) (string, string) {
+	namespace := ""
+	name := ""
+
+	if idx := strings.Index(vsv, "."); idx > 0 {
+		name = vsv[:idx]
+		namespace = vsv[idx+1:]
```

Comment:
I dont think this will work for fully qualified virtualservice names which have multiple `.`. e.g:

```
my-vsvc.my-namespace.svc.cluster.local
```

Namespace will be parsed as `my-namespace.svc.cluster.local` which won't work to subsequent K8s API call

Reply:
Thanks. You are right. I switched to using `strings.Split` in the updated code.